hpy/tools/autogen/parse.py

```diff
@@ -202,16 +203,26 @@ class HPyAPI:
                             re.DOTALL | re.MULTILINE)
 
     def __init__(self, filename):
-        cpp_cmd = get_config_var('CC').split(' ')
+        cpp_cmd = get_config_var('CC')
+        if cpp_cmd:
+            cpp_cmd = cpp_cmd.split(' ')
+        elif sys.platform == 'win32':
```

Comment:
Interesting. Is conf variable `CC`not set on Windows? I did it this way assuming that we would get `cl.exe` from `CC. I thought I've tested that on Windows but since it's not tested in the CI, I can imagine that this changed or is different on other Windows versions.

Reply:
CC is empty on Windows - perhaps because finding MSVC has been a pain nobody bothered with? Even in distutils it was a bit of an unreliable mess.